freeciv-web/build.sh

```diff
@@ -24,4 +24,5 @@ mkdir -p $DATADIR/scorelogs
 mkdir -p $DATADIR/ranklogs
 
 echo "maven package"
+sudo mvn compile flyway:migrate
 mvn package && cp target/freeciv-web.war "${TOMCATDIR}/webapps/ROOT.war"
```

Comment:
I'm not sure that this change should be added, since it will slow down building.

Reply:
I know, it will slow down building. I introduced this step, for the DB migration steps to kick-in. Otherwise the relevant mysql tables will not be available and all servlets (e.g., meta server), will not work. At least to my understanding of how those components interact.